omniscidb/ArrowStorage/ArrowStorage.cpp

```diff
@@ -470,14 +470,19 @@ void ArrowStorage::appendArrowTable(std::shared_ptr<arrow::Table> at, int table_
   for (auto& frag : fragments) {
     frag.metadata.resize(at->columns().size());
   }
-
   mapd_shared_lock<mapd_shared_mutex> dict_lock(dict_mutex_);
   threading::parallel_for(
       threading::blocked_range(0, (int)at->columns().size()), [&](auto range) {
         for (auto col_idx = range.begin(); col_idx != range.end(); col_idx++) {
           auto col_info = getColumnInfo(db_id_, table_id, col_idx + 1);
           auto col_type = col_info->type;
           auto col_arr = at->column(col_idx);
+          LOG(INFO) << "append Arrow: " << col_type->toString()
```

Comment:
INFO is a default logging level. I don't think this message should go to the log by default. Also, it lacks log context because we don't have much logging in the ArrowStorage, and therefore is not very useful. If it was only for your personal debugging, then remove it.

Reply:
Ok, I will remove it, but I think there should be more debug logging. Arrow storage types and issues during append and types conversion not logged.